### PR TITLE
Frequently Asked Questions

### PR DESCRIPTION
**Pull request by feitnomore**
As documentation is still in very initial phase, I thought it would be nice to have a little FAQ for main questions I've got around the tool when testing it myself.

Thanks

**What this PR does / why we need it**: This just adds a simple FAQ document, so that the project can add documentation in an easy way while the official documentation is not ready.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    {
+      "Network": "10.243.0.0/16",
+      "Backend": {
+        "Type": "vxlan",
+        "Port": 1234
+      }
+    }
+```
+
+## Whenever I reboot a VM in Kubevirt, it looses is IP address, how do I fix it?
```
Comment (davidvossel) [inline docs/faq.md:32]: looses/loses

also, this shouldn't be a problem. restarting a vm will result in a new ip, but i believe kubernetes is resilient to nodes having a different ip after restart.

does this for sure cause an issue?

Comment (feitnomore) [reply docs/faq.md:32]: image

This is what I was trying to document here.

Thanks

Comment (davidvossel) [reply docs/faq.md]: ah yes, ha, i remember that now. I had experimented with worker nodes receiving new IP addresses after restart with some success, but it is true that control plane nodes offer a different set of challenges

**Review by davidvossel — APPROVED** [review]
/lgtm
/approve

**Pull request merged**